cognite/neat/rules/models/wrapped_entities.py

```diff
@@ -15,6 +17,7 @@ class WrappedEntity(BaseModel, ABC):
     name: ClassVar[str]
     _inner_cls: ClassVar[type[Entity]]
     inner: list[Entity] | None
+    filter: str | None = None
```

Comment:
This seems to be only used by the RawFilter, so is there any reason to have it on the base class?
```suggestion
```